frontend/dist/index.mak

```diff
@@ -27,6 +27,8 @@
         param = "color"
         color_change_value = 15
       %>
+      :root { --primaryColor: ${primary_color_str }}
+
```

Comment:
Will this work with servdev? AFAIK they do not use this file.

Reply:
Oh I finally do not use it. I should remove this.